_data/simple-icons.json

```
@@ -15,6 +15,11 @@
             "hex": "00A98F",
             "source": "https://about.me/assets"
         },
+        {
+            "title": "ABB Robotstudio",
```

Given the title of [the page you linked](https://new.abb.com/products/robotics/en/robotstudio/downloads) I would expect the capatalization to be as "RobotStudio" 🤔 

```suggestion
            "title": "ABB RobotStudio",
```

Remember to update it in the SVG as well 😉